services/bettersocial/user/BetterSocialBlockUserV2.js

```diff
@@ -89,6 +89,7 @@ const BetterSocialBlockUserV2 = async (token, selfUserId, targetUserId, source,
   }
 
   BetterSocialScoreBlockUser(selfUserId, targetUserId, postId);
+  BetterSocialScoreBlockUser(selfUserId, targetAnonymousUserId?.user_id, '');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new call to `BetterSocialScoreBlockUser` has been added, passing `targetAnonymousUserId?.user_id` and an empty string. Ensure that the function can handle potential undefined values for `targetAnonymousUserId?.user_id` and that it behaves as expected when passed an empty string.
```diff
+ BetterSocialScoreBlockUser(selfUserId, targetAnonymousUserId?.user_id, '');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->